addons/binding/org.openhab.binding.oceanic/src/main/java/org/openhab/binding/oceanic/OceanicBindingConstants.java

```diff
@@ -0,0 +1,314 @@
+/**
+ * Copyright (c) 2014 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.oceanic;
+
+import java.io.InvalidClassException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.types.Type;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link OceanicBinding} class defines common constants, which are used
+ * across the whole binding.
+ *
+ * @author Karel Goderis - Initial contribution
+ */
+public class OceanicBindingConstants {
+
+    public static final String BINDING_ID = "oceanic";
+
+    // List of all Thing Type UIDs
+    public final static ThingTypeUID THING_TYPE_SOFTENER = new ThingTypeUID(BINDING_ID, "softener");
+
+    // List of all Channel ids
+    public enum OceanicChannelSelector {
+
+        getSRN("serial", StringType.class, ValueSelectorType.GET, true),
```

Comment:
serial, mac, name and type are now properties (which are never set...?), so I assume they should be removed from the ChannelSelector?